topology/src/main/java/io/camunda/zeebe/topology/metrics/TopologyMetrics.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.topology.metrics;
+
+import io.camunda.zeebe.topology.state.ClusterChangePlan;
+import io.camunda.zeebe.topology.state.ClusterChangePlan.Status;
+import io.camunda.zeebe.topology.state.ClusterTopology;
+import io.camunda.zeebe.topology.state.CompletedChange;
+import io.prometheus.client.Enumeration;
+import io.prometheus.client.Gauge;
+import java.time.Instant;
+import java.util.List;
+
+public final class TopologyMetrics {
+  private static final String NAMESPACE = "zeebe";
+  private static final Gauge TOPOLOGY_VERSION =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_topology_version")
+          .help("The version of the cluster topology")
+          .register();
+  private static final Gauge CHANGE_ID =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_id")
+          .help("The id of the cluster topology change plan")
+          .register();
+  private static final Enumeration CHANGE_STATUS =
+      Enumeration.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_status")
+          .help("The state of the current cluster topology")
+          .states(ClusterChangePlan.Status.class)
+          .register();
+  private static final Gauge CHANGE_VERSION =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_version")
+          .help("The version of the cluster topology change plan")
+          .register();
+
+  private static final Gauge PENDING_OPERATIONS =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_operations_pending")
+          .help("Number of pending changes in the current change plan")
+          .register();
+  private static final Gauge COMPLETED_OPERATIONS =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_operations_completed")
+          .help("Number of completed changes in the current change plan")
+          .register();
+  private static final Gauge STARTED_AT =
+      Gauge.build()
+          .name(NAMESPACE)
+          .name("cluster_changes_started_at")
+          .help("Time at which the current topology change was started")
+          .register();
+  private static final Gauge COMPLETED_AT =
+      Gauge.build()
+          .namespace(NAMESPACE)
+          .name("cluster_changes_completed_at")
+          .help("Time at which the current topology change was completed")
+          .register();
```

Comment:
:thought_balloon: Do we really need these ( started_at and completed_at) in the metrics? Or just a metrics for duration of how long a topology change lasted? We can always query the actuator to find the actual timestamps.

Reply:
I think I'll remove those for now :+1: